src/nist.py

```diff
@@ -66,4 +67,84 @@ def get_NIST_ids():
         'T1 - NIST sphere 13',
         'T1 - NIST sphere 14',
     ]
-    return ids
\ No newline at end of file
+    return ids
+
+def temperature_correction(input_temperature):
```

Comment:
We should add another input for the serial number, because the values in your table are only valid for serial_number>=0042. If the serial number is less than that, we should output None maybe and output a warning saying that this functionality isn't there yet

Reply:
Done!